src/reducers/provider-state-updaters.js

```diff
@@ -48,6 +49,7 @@ import KeplerGlSchema from 'schemas';
 
 export const INITIAL_PROVIDER_STATE = {
   isProviderLoading: false,
+  isCloudMapLoading: false,
```

Comment:
why can't we just set `isProviderLoading`? looks like in all places these 2 values are the same

Reply:
I tried, but it is used when getting maps list too and it enters in an endless loop and I preferred not to refactor it for tomorrow as it may affect the rest of map loading methods.